test/jquery.ime.test.fixtures.js

```diff
@@ -4127,5 +4127,17 @@ var palochkaVariants = {
 			{ input: [ 'v', [ '1', true ], 'i', 'T', 'U' ], output: 'វ‌ិទូ', description: 'ZWNJ (savant): v AltGr-1 iTU -> វ‌ិទូ' },
 			{ input: [ 'v', [ '`', true ], 'i', 'T', 'U' ], output: 'វ‍ិទូ', description: 'ZWJ (savant): v AltGr-` iTU -> វ‍ិទូ' }
 		]
+	},
+	{
+		description: 'fon alt test',
```

Comment:
I see that you renamed the input method to fon tilde, should not we change this description too?